src/reuse/modules/mobile/android.ts

```diff
@@ -0,0 +1,91 @@
+import { Element } from "../../../../@types/wdio";
+import { VerboseLoggerFactory } from "../../helper/verboseLogger";
+import ErrorHandler from "../../helper/errorHandler";
+/**
+ * @class Android
+ * @memberof Mobile
```

Comment:
I guess the @memberof value has to be lowercase since the final namespaces are always lowercase. Otherwise the docs will not be mapped correctly.
```suggestion
 * @memberof mobile
```

Reply:
same everywhere

Reply:
we can do the changes in our call and verify the docs together